README.adoc

```diff
@@ -711,7 +748,7 @@ Tests run: 2, Failures: 0, Errors: 0, Skipped: 0
 
 == Tearing down chart releases
```

Comment:
For Windows/Mac you'll also want to delete the NGINX ingress controller.

Reply:
Fixed